.travis.yml

```diff
@@ -42,6 +42,7 @@ script:
   - cargo build --verbose --features llvm_stable
   - make test
   - git add -A
+  - git diff @
   - git diff-index --quiet HEAD
```

Comment:
No, git diff-index only gives an error on empty diffs if run with --quiet.

(It also outputs object ids, not diffs)